src/lib/comms/Makefile

```diff
@@ -121,12 +124,21 @@ obj-networking-$(MAVLINK)-extra-cflags += \
 obj-networking-$(LWM2M) += \
 	sol-lwm2m.o
 
+obj-bluetooth-$(BLUETOOTH) += \
+	sol-bluetooth.o
+
+obj-bluetooth-$(PLATFORM_LINUX) += \
```

Comment:
should we do "bluez" instead of linux here? I wonder if some crazy dudes may come with their own "small bluetooth"  one day. Calling it bluez is more specific and clear on what we do. Same for `impl` filename